docs/item.rst

```diff
@@ -0,0 +1,93 @@
+item
+====
+
+.. dfhack-tool::
+    :summary: Perform bulk operations on items based on various properties.
+    :tags: fort productivity items
+
+Filter items in you fort by various properties (e.g., item type, material,
+wear-level, quality, ...), and perform the bulk operations forbid, dump, melt,
+and their inverses. By default, the tool does not consider artifacts. Outputs
+the number of items that matched the filters and were modified.
+
+Usage
+-----
+
+``item [ help | count | [un]forbid | [un]dump | [un]melt ] <filter option>``
+
+Action names should be self explanatory. All actions other than ``help`` and
+``count`` have implicit filters associated with them. For instance, ``item
+forbid --unreachable`` will neither report nor test reachability of items that
+are already forbidden.
+
+Examples
+--------
+
+``item forbid --unreachable``
+    Forbid all items that cannot be reached by any of your citizens.
+
+``item unforbid --inside Cavern1 --type wood``
+    Unforbid/reclaim all logs inside the burrow named "Cavern1" (Hint: use 3D
+    flood-fill to create a burrow covering an entire cavern layer)
+
+``item melt -t weapon -m steel --max-quality 3``
+    Designate all steel weapons whose quality is less than "exceptional" for
+    melting.
+
+Options
+-------
+
+``-n, --dry-run``
+    Use get an accurate count of the items that would be affected, including the
+    implicit filters of the selected main action.
```

Comment:
the implicit filters need documenting : )

Reply:
There is already a paragraph right under "Usage". Are you saying that's not enough?

Reply:
I'm not sure I'd call not reporting items that are already forbidden an "implicit filter". If `item forbid --unreachable` included the count of items that were already forbidden, I'd just call that a bug : P

What about items that are owned, in a job, rotten, on fire, part of a building, etc.? Many other tools implicitly filter these out. I thought this was more what you meant.

Reply:
after reading the actual code, I see what you meant by "implicit filter" I suggested some rewording to make it (I think) clearer